dbms/src/Functions/FunctionsString.cpp

```diff
@@ -42,13 +44,17 @@ struct EmptyImpl
         }
     }
 
-    static void vector_fixed_to_constant(const ColumnString::Chars_t & /*data*/, size_t n, UInt8 & res)
+    static void vector_fixed_to_constant(const ColumnString::Chars_t & /*data*/, size_t /*n*/, UInt8 & /*res*/)
     {
-        res = negative ^ (n == 0);
     }
 
-    static void vector_fixed_to_vector(const ColumnString::Chars_t & /*data*/, size_t /*n*/, PaddedPODArray<UInt8> & /*res*/)
+    static void vector_fixed_to_vector(const ColumnString::Chars_t & data, size_t n, PaddedPODArray<UInt8> & res)
     {
+        char * empty_chars[n] = {0};
```

Comment:
MAX_FIXEDSTRING_SIZE is 16 MB and this code can lead to stack overflow
(16 MB is impractical size for FixedString, but malicious user can pass this value to kill the server.)

Reply:
And missing `const`.

Reply:
And it doesn't compile with clang 7.